modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/model/SupportedLanguage.java

```diff
@@ -0,0 +1,10 @@
+package com.liferay.translation.translator.deepl.internal.model;
+
+/**
+ * @author Yasuyuki Takeo
+ */
+public class SupportedLanguage {
+  public String language;
+  public String name;
+  public Boolean supports_formality;
```

Comment:
Can you make this class a full POJO? Add getters and setters and make the fields private (JSON serialization should work anyway).